ui/src/app/kvm/views/KVMList/KVMListTable/OSColumn/OSColumn.tsx

```diff
@@ -13,37 +13,44 @@ import DoubleRow from "app/base/components/DoubleRow";
 
 type Props = { id: number };
 
-const OSColumn = ({ id }: Props): JSX.Element => {
+const OSColumn = ({ id }: Props): JSX.Element | null => {
   const pod = useSelector((state: RootState) =>
     podSelectors.getById(state, id)
   );
-  const host = useSelector((state: RootState) =>
+  const hostDetails = useSelector((state: RootState) =>
     podSelectors.getHost(state, pod)
   );
   const osReleases = useSelector((state: RootState) =>
-    generalSelectors.osInfo.getOsReleases(state, host && host.osystem)
+    generalSelectors.osInfo.getOsReleases(
+      state,
+      hostDetails && hostDetails.osystem
+    )
   );
   const machinesLoading = useSelector(machineSelectors.loading);
   const controllersLoading = useSelector(controllerSelectors.loading);
-  const loading = machinesLoading || controllersLoading;
 
-  let osText = "Unknown";
-  if (host) {
-    osText = getStatusText(host, osReleases);
-  } else if (!host && loading) {
-    osText = "";
-  }
+  if (pod) {
+    const loading =
+      Boolean(pod.host) &&
```

Comment:
why the `Boolean` cast here? That seems a bit unidiomatic.

Reply:
Another strict mode change. `loading` is meant to be a boolean, but if it short circuits here it actually becomes a string (`pod.host`) which isn't right. I prefer this method over `!!pod.host` which is a bit cryptic if you're not familiar with javascript.

Reply:
I think maybe we should consider turning on https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/strict-boolean-expressions.md which prevents any values in boolean expressions being anything other than `boolean` or `never`. What do you think?